src/app/pages/storage/components/dashboard-pool/disk-health-card/disk-health-card.component.ts

```diff
@@ -76,7 +76,7 @@ export class DiskHealthCardComponent implements OnInit, OnChanges {
   }
 
   get iconType(): PoolCardIconType {
-    if (this.diskState.alerts || this.diskState.smartTests) {
+    if (this.diskState.smartTests) {
```

Comment:
By suppressing the icon for all alerts, it also won't show when there are temperature related alerts.
We should probably be trying to filter the test status instead

Reply:
I totally agree, thanks for noticing that